chaos_wrapper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function rollback() {
+    # avoid commits from the same faulty PR
+    rb_commit=$(git log --author Chaos --format=format:%H | tail -n +2 | head -n 1)
```

Comment:
Yeah, I know that, but unless we want chaosbot GPG signing commits there's no other way. In addition, we can spot PRs with commits with an author of Chaos.